src/main/java/by/onliner/cucumber/steps/MainSteps.java

```diff
@@ -0,0 +1,28 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static by.onliner.core.utils.EqualsUtil.equalContains;
+
+public class MainSteps extends AbstractSteps{
+
+    @When("Click [О компании] link")
+    public void clickOnAboutCompanyLink() {
+        mainPage.getFooter().clickOnAboutCompanyLink();
+    }
+
+    @Then("Page url should contain [about]")
+    public void pageUrlShouldContainAbout() {
```

Comment:
I think it's worth passing the expected result as an argument and so that you will be able to reuse this method for other pages.

Reply:
Realized